### ISSUE TITLE
can not highlight code

### ISSUE THREAD

**Issue by wliangde** [BoostIO/Boostnote#3042]
# Current behavior
The newest version:Boostnote 0.11.17  can not highlight code.

# Expected behavior
highlight code

# Steps to reproduce

1. make a new markdown file
2.  write
```
\```js
var code="wld";
\```
```
3. 

# Environment

- Version :Boostnote 0.11.17
- OS Version and name : windows 64

**Comment by Bubka** [comment]
Hi,

Same bug on my side, since 0.11.16
The `railcasts.css` file cannot be loaded because the path to the file is partially duplicated in the js var. See my chrome logs:

#### The request headers:
```
Request URL: file:///C:/Users/xx123456/AppData/Local/boost/app-0.11.17/resources/app/C:/Users/xx123456/AppData/Local/boost/app-0.11.17/resources/app/node_modules/codemirror/theme/railscasts.css
Referrer Policy: no-referrer-when-downgrade
```

#### in `main.js`:

![image](https://user-images.githubusercontent.com/858858/58423067-82d4a580-8094-11e9-80ea-24fa2c1ffc5f.png)

#### My console.log
```
DevTools failed to parse SourceMap: file:///C:/Users/xx123456/AppData/Local/boost/app-0.11.17/resources/app/node_modules/@rokt33r/js-sequence-diagrams/dist/sequence-diagram.js
main.js:2436 GET file:///C:/Users/xx123456/AppData/Local/boost/app-0.11.17/resources/app/C:/Users/xx123456/AppData/Local/boost/app-0.11.17/resources/app/node_modules/codemirror/theme/railscasts.css 0 ()
get @ main.js:2436
(anonymous) @ main.js:5416
__webpack_require__ @ warning.es6:11
(anonymous) @ main.js:17995
__webpack_require__ @ warning.es6:11
module.exports.Object.defineProperty.value @ main.js:71784
__webpack_require__ @ warning.es6:11
module.exports.Object.defineProperty.value @ main.js:68924
__webpack_require__ @ warning.es6:11
module.exports.Object.defineProperty.value @ main.js:72784
__webpack_require__ @ warning.es6:11
module.exports.module.exports @ warning.es6:38
__webpack_require__ @ warning.es6:11
(anonymous) @ warning.es6:31
(anonymous) @ warning.es6:32
main.js:33810 GET file:///C:/Users/xx123456/AppData/Local/boost/app-0.11.17/resources/app/C:/Users/xx123456/AppData/Local/boost/app-0.11.17/resources/app/node_modules/codemirror/theme/railscasts.css 0 ()
applyStyle @ main.js:33810
componentDidMount @ main.js:33716
e.notifyAll @ react-dom.min.js:10
close @ react-dom.min.js:12
closeAll @ react-dom.min.js:13
perform @ react-dom.min.js:13
perform @ react-dom.min.js:13
perform @ react-dom.min.js:12
k @ react-dom.min.js:12
closeAll @ react-dom.min.js:13
perform @ react-dom.min.js:13
batchedUpdates @ react-dom.min.js:12
u @ react-dom.min.js:12
r @ react-dom.min.js:12
enqueueSetState @ react-dom.min.js:12
r.setState @ react.min.js:10
(anonymous) @ main.js:206506
invokeFunc @ C:\Users\xx123456\AppData\Local\boost\app-0.11.17\resources\app\node_modules\lodash\lodash.js:10333
trailingEdge @ C:\Users\xx123456\AppData\Local\boost\app-0.11.17\resources\app\node_modules\lodash\lodash.js:10382
timerExpired @ C:\Users\xx123456\AppData\Local\boost\app-0.11.17\resources\app\node_modules\lodash\lodash.js:10370
setTimeout (async)
setTimeout @ C:\Users\xx123456\AppData\Local\boost\app-0.11.17\resources\app\node_modules\lodash\lodash.js:6646
leadingEdge @ C:\Users\xx123456\AppData\Local\boost\app-0.11.17\resources\app\node_modules\lodash\lodash.js:10341
debounced @ C:\Users\xx123456\AppData\Local\boost\app-0.11.17\resources\app\node_modules\lodash\lodash.js:10410
componentWillReceiveProps @ main.js:206481
updateComponent @ react-dom.min.js:11
receiveComponent @ react-dom.min.js:11
receiveComponent @ react-dom.min.js:12
updateChildren @ react-dom.min.js:11
_reconcilerUpdateChildren @ react-dom.min.js:12
_updateChildren @ react-dom.min.js:12
updateChildren @ react-dom.min.js:12
_updateDOMChildren @ react-dom.min.js:11
updateComponent @ react-dom.min.js:11
receiveComponent @ react-dom.min.js:11
receiveComponent @ react-dom.min.js:12
updateChildren @ react-dom.min.js:11
_reconcilerUpdateChildren @ react-dom.min.js:12
_updateChildren @ react-dom.min.js:12
updateChildren @ react-dom.min.js:12
_updateDOMChildren @ react-dom.min.js:11
updateComponent @ react-dom.min.js:11
receiveComponent @ react-dom.min.js:11
receiveComponent @ react-dom.min.js:12
_updateRenderedComponent @ react-dom.min.js:11
_performComponentUpdate @ react-dom.min.js:11
updateComponent @ react-dom.min.js:11
receiveComponent @ react-dom.min.js:11
receiveComponent @ react-dom.min.js:12
_updateRenderedComponent @ react-dom.min.js:11
_performComponentUpdate @ react-dom.min.js:11
updateComponent @ react-dom.min.js:11
performUpdateIfNecessary @ react-dom.min.js:11
performUpdateIfNecessary @ react-dom.min.js:12
s @ react-dom.min.js:12
perform @ react-dom.min.js:13
perform @ react-dom.min.js:13
perform @ react-dom.min.js:12
k @ react-dom.min.js:12
closeAll @ react-dom.min.js:13
perform @ react-dom.min.js:13
batchedUpdates @ react-dom.min.js:12
u @ react-dom.min.js:12
r @ react-dom.min.js:12
enqueueSetState @ react-dom.min.js:12
r.setState @ react.min.js:10
a.handleChange @ react-redux.min.js:1
a @ redux.min.js:1
(anonymous) @ main.js:69130
Promise.then (async)
componentDidMount @ main.js:69129
e.notifyAll @ react-dom.min.js:10
close @ react-dom.min.js:12
closeAll @ react-dom.min.js:13
perform @ react-dom.min.js:13
s @ react-dom.min.js:12
perform @ react-dom.min.js:13
batchedUpdates @ react-dom.min.js:12
a @ react-dom.min.js:12
_renderNewRootComponent @ react-dom.min.js:12
_renderSubtreeIntoContainer @ react-dom.min.js:12
render @ react-dom.min.js:12
module.exports.Object.defineProperty.value @ main.js:72906
__webpack_require__ @ warning.es6:11
module.exports.module.exports @ warning.es6:38
__webpack_require__ @ warning.es6:11
(anonymous) @ warning.es6:31
(anonymous) @ warning.es6:32
```

**Comment by Bubka** [comment]
See 6012fc929ed11939918afec7fdb236dfd68279b7 that introduced recent changes on theme management

**Comment by MrClyfar** [comment]
I installed v0.11.16 today and encountered this very issue.

Each IDE theme I pick is not applied, the text appears black on a white background. Also, I had to re-install Boostnote as one time I opened the app, it was in a continous loop at startup. I do not know for certain if that was related to the theme issue I've experienced though.

**Comment by random34** [comment]
Same here - v0.11.17

The default theme works for me but all other preview code block theme seems to be broken.

**Comment by ZeroX-DG** [comment]
Possibly related https://github.com/BoostIO/Boostnote/issues/3018

**Comment by Flexo013** [comment]
Fixed in next release v0.12.0.

https://github.com/BoostIO/Boostnote/pull/3048 (merged)
